pages/templates/pages/group_page.html

```diff
@@ -15,7 +15,7 @@
 
       <div class="container pt6">
         <p class="mb0">
-          {% if group.group_type != 'state-organizing-committee' %}
+          {% if group.group_type != 1 %}
```

Comment:
👍  I suggest adding code comment too for this section.

Reply:
Thanks, although I meant more like "Show different content for SOC vs whatever". More like to explain the logic in general here.